discovery-provider/scripts/start.sh

```diff
@@ -2,7 +2,9 @@
 
 if [[ -z "$audius_loggly_disable" ]]; then
     if [[ -n "$audius_loggly_token" ]]; then
+        audius_discprov_hostname=$(echo $audius_discprov_url | sed -e 's/[^/]*\/\/\([^@]*@\)\?\([^:/]*\).*/\2/')
```

Comment:
can you comment the intent of this regex?

Reply:
Done, thanks!